packages/core/src/Popout/Popout.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { RefObject, useCallback, useEffect, useLayoutEffect, useRef, useState } from 'react'
+import styled from 'styled-components'
+import { Box } from '..'
+
+export interface IPopoutProps {
+  trigger: JSX.Element
+  content?: JSX.Element | undefined
+  triggerRef?: RefObject<HTMLElement>
+  onOpen?: () => void
+  onClose?: () => void
+  closeOnTriggerRefClick?: boolean
+}
+
+const TriggerWrapper = styled(Box)`
+  position: relative;
+  width: fit-content;
+`
+
+const OnClickHandler = styled(Box)`
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  cursor: pointer;
+`
+
+const Shadow = styled(Box)`
+  position: fixed;
+  top: 0;
+  left: 0;
+  cursor: pointer;
+  width: 100vw;
+  height: 100vh;
+  z-index: 1;
+  transition: opacity 250ms ease-in-out;
+`
+
+const PopoutModal = styled(Box)`
+  position: absolute;
+  z-index: 2;
+  background-color: white;
```

Comment:
Can you update to use `bg='background.lightest'` ?

Reply:
Fixed this, great catch!